### PR TITLE
docs: `integrations` reference updates 13

### PR DESCRIPTION
**Pull request by leo-gan**
Added missed provider pages and links. Fixed inconsistent formatting.
Added arxiv references to docstirngs.

### CODE REVIEW

File: docs/docs/integrations/providers/qdrant.mdx [hunk 1]
```diff
@@ -13,6 +13,27 @@ Install the Python partner package:
...
 ```
 
+## Embedding models
+
+### FastEmbedEmbeddings
+
+See a [usage example](/docs/integrations/text_embedding/fastembed).
+
+```python
+from langchain_community.embeddings.fastembed import FastEmbedEmbeddings
```
Comment (efriis) [inline docs/docs/integrations/providers/qdrant.mdx:23]: remove and happy to deprecate in favor of langchain_qdrant ones

File: libs/community/langchain_community/embeddings/fastembed.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
 class FastEmbedEmbeddings(BaseModel, Embeddings):
```
Comment (efriis) [inline libs/community/langchain_community/embeddings/fastembed.py:13]: let's deprecate this

File: docs/docs/integrations/providers/friendly.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Friendli AI
+
+>[Friendli AI](https://fireworks.ai/) is company that fine-tunes, deploys LLMs,
```
Comment (efriis) [inline docs/docs/integrations/providers/friendly.md:3]: fix link

File: docs/docs/integrations/providers/qdrant.mdx (hunk 1 above)
Comment (leo-gan) [reply docs/docs/integrations/providers/qdrant.mdx:23]: Yeah, I do not add description of classes that marked with the "deprecated" decorator. This class was not attributed with the decorator, so I was confused :)

File: docs/docs/integrations/providers/friendly.md (hunk 3 above)
Comment (leo-gan) [reply docs/docs/integrations/providers/friendly.md:3]: Good catch! TNX!

File: libs/community/langchain_community/embeddings/fastembed.py (hunk 2 above)
Comment (leo-gan) [reply libs/community/langchain_community/embeddings/fastembed.py:13]: @baskaryan ?

Comment (efriis) [reply libs/community/langchain_community/embeddings/fastembed.py:13]: how was this resolved?

Comment (leo-gan) [reply libs/community/langchain_community/embeddings/fastembed.py:13]: Yes, I removed FastEmbedEmbeddings class reference from qdrant.mdx page.
If you mean the deprecate decorator for this class, I assume this change should be implemented in another PR (this PR is only about doc changes).

**Review by efriis — APPROVED** [review]

**Comment by leo-gan** [comment]
@ccurme Can you, please, copy-paste the Vercel error? I don't have access to it and cannot fix it. :( Thanks!

**Comment by efriis** [comment]
some sort of content hashing problem

ScreenShot 2024-08-30 at 02 07 10PM

**Comment by leo-gan** [comment]
@efriis I was trying to rerun checks with rebasing merged PRs, but I had no luck.

**Comment by leo-gan** [comment]
@efriis 
Is that possible that checks failed on this (in templates/rag-aws-bedrock/README.md)?

> 
> ```shell
> langchain serve
> ```
> 
> This will start the FastAPI app with a server running locally at [http://localhost:8000](http://localhost:8000)
> 
> You can see all templates at [http://127.0.0.1:8000/docs](http://127.0.0.1:8000/docs) and access the playground at
>[http://127.0.0.1:8000/rag-aws-bedrock/playground](http://127.0.0.1:8000/rag-aws-bedrock/playground).

**Comment by leo-gan** [comment]
@hwchase17 Thanks!

**Pull request merged**